admin_guide/allocating_node_resources.adoc

```diff
@@ -207,21 +211,13 @@ $ curl <certificate details> https://<master>/api/v1/nodes/cluster.node22/proxy/
 See xref:../rest_api/index.adoc#rest-api-index[REST API Overview] for more details about certificate details.
 
 [[node-enforcement-1]]
-== Node enforcement
+== Node Enforcement
 
-The node is able to limit the total amount of resources that pods
-may consume based on the configured allocatable value.  This feature significantly
-improves the reliability of the node by preventing pods from starving
-system services (for example: container runtime, node agent, etc.) for resources.
-It is strongly encouraged that administrators reserve
-resources based on the desired node utilization target
-in order to improve node reliability.
+The node is able to limit the total amount of resources that pods can consume based on the configured allocatable value. This feature significantly improves the reliability of the node by preventing pods from using CPU and memory resources that are needed by system services such as the container runtime and node agent. It is strongly encouraged that administrators reserve resources based on the desired node utilization target in order to improve node reliability.
```

Comment:
Is "node utilization target" an actual term? If not, ISG suggests s/utilization/use